Dockerfile

```diff
@@ -61,20 +61,20 @@ RUN find /opt/vagrant/embedded/ -type f | grep -v /opt/vagrant/embedded/plugins.
 
 RUN /opt/vagrant/embedded/bin/gem install --install-dir /opt/vagrant/embedded/gems/${VAGRANT_VERSION} ./pkg/vagrant-libvirt*.gem
 
-RUN cat <<EOF > /opt/vagrant/embedded/plugins.json
-{
-    "version": "1",
-    "installed": {
-        "vagrant-libvirt": {
-            "ruby_version":"$(/opt/vagrant/embedded/bin/ruby -e 'puts RUBY_VERSION')",
-            "vagrant_version":"${VAGRANT_VERSION}",
-            "gem_version":"",
-            "require":"",
-            "sources":[]
-        }
-    }
-}
-EOF
+RUN echo -n '{\n\
+    "version": "1",\n\
+    "installed": {\n\
+        "vagrant-libvirt": {\n\
+            "ruby_version":"' > /opt/vagrant/embedded/plugins.json \
```

Comment:
Instead of outputting to a file here, does the following work?
```
        "ruby_version": "'$(/opt/vagrant/embedded/bin/ruby -e 'puts "#{RUBY_VERSION}"')'",\n\
        "vagrant_version": "'${VAGRANT_VERSION}'",\n\
            "gem_version":"",\n\
            "require":"",\n\
            "sources":[]\n\
        }\n\
    }\n\
}' >> /opt/vagrant/embedded/plugins.json
```

Basically making use of `'` to switch back to allowing shell expansion and then re-opening with `'` for the echo to continue.

Reply:
unfortunately shell expansion does not work in between `'` braces

Reply:
The change above seems to work when I tried it with buildah locally, the `'` are being closed temporarily to allow the next piece use shell expansion and then re-opened again.

Reply:
Ah, indeed! Sorry, I did not read your comment correctly. I've used your changes and pushed them.